resource_customizations/cdi.kubevirt.io/DataVolumes/health.lua

```diff
@@ -0,0 +1,12 @@
+hs = { status="Progressing", message="No status available"}
```

Comment:
We can supplement the phase with the Running condition to try and expose errors:
- Succeeded --> Healthy
- Failed --> Degraded
- Unknown --> Unknown
- Paused --> Suspended
- Other phase + Running --> Progressing
- Other phase + !Running --> Degraded